### PR TITLE
Disables player deadites and removes rotmen. Fixes cure rot.

### PR DESCRIPTION
**Pull request by Gyrandola**
## About The Pull Request

title.

## Why It's Good For The Game

cure rot actually works now. player deadites are once again disabled because theres basically zero rp you can do as a zombie.

### CODE REVIEW

**Comment by katsukai** [comment]
Thank fuck, if you can get this fixed.

File: code/modules/spells/roguetown/acolyte/pestra.dm [hunk 1]
```diff
@@ -168,17 +170,22 @@
 					has_rot = TRUE
 					break
 		if(!has_rot)
-			to_chat(user, span_warning("Nothing happens."))
+			to_chat(user, "<span class='warning'>Nothing happens.</span>")
 			return FALSE
 		if(GLOB.tod == "night")
-			to_chat(user, span_warning("Let there be light."))
+			to_chat(user, "<span class='warning'>Let there be light.</span>")
```
Comment (Cyprex) [inline code/modules/spells/roguetown/acolyte/pestra.dm:176]: No, these exist for a reason, and are a good thing. Revert these two lines

**Comment by Gyrandola** [comment]
Appreciate the effort but whatever changes you did, they didnt fix the only problem left before I went to sleep, which is zombie traits not being removed on rot removal

**Comment by Nakalon** [comment]
Hows this behaving Gyran?

**Comment by Gyrandola** [comment]
> Hows this behaving Gyran?

seems to be working

**Comment by Nakalon** [comment]
> > Hows this behaving Gyran?
> 
> seems to be working

Want it in?

**Comment by SeizingSponge** [comment]
I don't play deadite, but I will say I highly disagree that deadite provides zero RP, because it SPAWNS RP. The player's playing the deadite create RP just by existing and showing up in different scenarios and locations. A player deadite can show up in very interesting locations and situations that an NPC deadite wouldn't be able to achieve. Also the fact that some deadite are intelligent gives a special feeling similar to special zombies in other games. My RP has been positively enhanced by other people playing deadite because they themselves are catalysts that cause other people to have to react to the event accordingly, which can add a lot more variance to RP scenarios.

**Comment by hugmeorelse** [comment]
Right, we're still having problems. The traits simply are not being removed when they're being de-rotted or revived. I don't remember this being an issue beforehand, so there's some fuckery afoot.

**Review by Nakalon — APPROVED** [review]
This thing is causing such a silly mess!

**Comment by Nakalon** [comment]
> I don't play deadite, but I will say I highly disagree that player deadites provide zero RP, because they SPAWN RP. The player's playing the deadite create RP just by existing and showing up in different scenarios and locations. A player deadite can show up in very interesting locations and situations that an NPC deadite wouldn't be able to achieve. Also the fact that some deadite are intelligent gives a feeling similar to encountering special zombies in other games. My RP has been positively enhanced by other people playing deadite because they themselves are catalysts that cause other people to have to react to the event accordingly, which can add a lot more variance to RP scenarios.

We're not removing it just because of this.
For what they provide, they're causing a lot of trouble, so we have to weigh our options!

**Comment by 2PlainGhoul** [comment]
People were literally turning Into Zombies and going to the church for free revives -- If players can't treat the role with respect than yeah; It should be fucking removed, honestly -- I can see the deadite coming back as some sorta antag? Like maybe at some point Zizo cultists will be able to turn people Into controllable zombies? just food for thought

**Comment by RadiantFlash7** [comment]
> > People were literally turning Into Zombies and going to the church for free revives -- If players can't treat the role with respect than yeah; It should be fucking removed, honestly -- I can see the deadite coming back as some sorta antag? Like maybe at some point Zizo cultists will be able to turn people Into controllable zombies? just food for thought
> 
> ahhhhh okay, I did not know this was happening. Is it possible to make people who have become deadites unrevivable? Like once your body has become a full on deadite, Zizo has already "claimed" you and there is no reversing it?

That defeats the point of being able to /cure rot/.

**Pull request merged**